cfgov/legacy/templates/complaint/complaint-landing.html

```diff
@@ -35,13 +35,147 @@
+        {% if ccdb_sept_2019_updates %}
+        <p class="hero_subhead">This database is a collection of complaints about consumer financial products and services that we sent to companies for response.</p>
+        {% else %}
         <p class="hero_subhead">Each week we send thousands of consumers’ complaints about financial products and services to companies for response. Those complaints are published here after the company responds or after 15 days, whichever comes first. By adding their voice, consumers help improve the financial marketplace.</p>
+        {% endif %}
       </div>
       <div class="hero_image" style="background-image:url({% static 'complaint/img/hero.png' %})"></div>
     </div>
   </section>
-  <div class="wrapper  wrapper__match-content content_wrapper">
+  <div class="wrapper wrapper__match-content content_wrapper">
     <div class="content_full">
+      {% if ccdb_sept_2019_updates %}
+      <div class="block block__bg block__flush-bottom block__border-top block__border-bottom">
+        <div class="content-l">
+          <div class="content-l_col content-l_col-2-3">
+            <h2 class="h3">
+              Things to know before you use the database
+            </h2>
+            <p>
+              Complaints are published after the company responds, confirming a commercial relationship with the consumer, or after 15 days, whichever comes first.
+              <a href="#about-the-database">
+                Learn more
+              </a>
+            </p>
+            <p>
+              Complaints are not necessarily representative of all consumers’ experiences with a financial product or company.
+              <a href="#what-you-should-consider">
+                Learn more
+              </a>
+            </p>
+            <p>
+              We don’t verify all the allegations in complaint narratives.
+              <a href="#what-you-should-consider">
+                Learn more
+              </a>
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="block">
+        <div class="section-one ccdb-landing_section content-l">
+          <div class="content-l_col content-l_col-1-3">
+            <img class="isocon" alt="Complaint data" src="{% static 'complaint/img/view.png' %}">
+            <p>View, sort, and filter data right in your browser.</p>
+            <div class="btn-holder">
+              <p><a href="/data-research/consumer-complaints/search/" class="btn">View complaint data</a></p>
+            </div>
+          </div>
+          <div class="content-l_col content-l_col-1-3">
+            <img class="isocon" alt="Download data and access the API" src="{% static 'complaint/img/download.png' %}">
+            <p>All complaint data we publish is freely available for anyone to use, analyze, and build on.</p>
+            <div class="btn-holder">
+              <p><a href="#download-the-data" class="btn">Download options and API</a></p>
```

Comment:
`<p>` needed?

Reply:
Same deal as above: Looks like there are some styles that target the `<p>`. I tried removing it, and everything looked the same in my quick check in Safari, but I didn't dig in too much or check in other browsers. So I'm hesitant to remove the `<p>` for now just in case given the tight timeline and all the eyes that will be on this page. But if we feel strongly about it, I can give it some more attention and can probably get it out of there.